README.md

```diff
@@ -0,0 +1,101 @@
+The correct answer here depends on our general release strategy for RAPIDS.
+
+## Dask
+
+Currently unless there is a breaking change in the HEAD of a dask repo that we cannot yet support, our typical strategy is to have dask pinnings that look something like `dask>=YYYY.MM.DD` in our package specs but then install the latest nightly (from source for pip, via the special dask nightly label in conda) for testing.
+Then, when we make a RAPIDS release we pin to the latest dask release.
+This bifurcated approach during development leads to an inaccurate statement of our dask dependency since in practice we cannot guarantee compatibility with anything except the HEAD of the `dask*` repos.
+Therefore, I propose that we eschew `>=` pinnings altogether for dask in this repo and instead always require the latest HEAD.
+We can do this by using a `git+https...` dependency on dask for pip, and for conda we can simply nest that inside a `pip:` section.
+That will ensure that our dependency is always consistent.
+Of course, if dask introduces a breaking change that will break our nightly packages as well, but that is no different from the current situation since ultimately any breaking change in dask means that users of our nightlies have to update to the latest nightlies of both `dask*` and RAPIDS packages.
+
+Concretely, the solution would look like replacing:
+```yaml
+  - name: rapids_dask_dependency
+    requirements:
+      run:
+        - dask >=2023.3.2
+        - dask-core >=2023.3.2
+        - distributed >=2023.3.2
+```
+with 
+```yaml
+  - name: rapids_dask_dependency
+    requirements:
+      run:
+        pip:
+            - git+https://github.com/dask/dask.git@main
+            - git+https://github.com/dask/distributed.git@main
+```
+and then reverting to a hard pin like `==2023.3.4` at release time.
+
+# Notes
+
+- I see no reasonable way to make pip and conda metapackage names align exactly, but I think that's fine.
+- For conda if we want to include a Python version dependency that will come in the form of a separate metapackage (whereas for pip it goes into the `requires-python` key of pyproject.toml.
+- After a release is completed, the `>=alpha` pinnings could cause problems because installing a nightly from a previous release would pull nightly dependencies corresponding to the current release. I do not think that this is a use case that we need to support, though, especially since we eventually purge our nightlies anyway
+- We could store these metapackages into a separate repo, or move them into the integration repo with our other metapackages. These metapackages are a bit different from those though since these are for our dependencies while those are for our dependents and users.
```

Comment:
```suggestion
- For conda if we want to include a Python version dependency that will come in the form of a separate metapackage, whereas for pip it goes into the `requires-python` key of pyproject.toml.
- After a release is completed, the `>=alpha` pinnings could cause problems because installing a nightly from a previous release would pull nightly dependencies corresponding to the current release. I do not think that this is a use case that we need to support, though, especially since we eventually purge our nightlies anyway.
- We could store these metapackages into a separate repo, or move them into the integration repo with our other metapackages. The metapackages included here are a bit different from those in the integration repo though since here they are for our dependencies while those from the integration repo are for our dependents and users.
```